src/oss-diff/DiffTool.cs

```diff
@@ -0,0 +1,437 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommandLine;
+using CommandLine.Text;
```

Comment:
capture color and reset if getting sigterm? I Ctrl+C'ed and now my terminal is red :) 
nit though

Reply:
Fixed